src/editors/EditorHeader.test.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import EditorHeader from './EditorHeader';
+import EditorPageContext from './EditorPageContext';
+import { ActionStates } from './data/constants';
+
+delete window.location;
+window.location = {
```

Comment:
You may want to be careful with doing this work at the top of the file like this, as it'll get evaluated when the file is loaded, not necessarily when the test is run.  You may want to instead put this into a `beforeEach` and then clean up after yourself afterward by putting the `assign` function back.

Reply:
I'll plan to clean these up! Thanks